src/config_parser.ml

```diff
@@ -94,20 +94,38 @@ type dhcp_range = {
   lease_time : int option;
 }
 
+type dhcp_config = {
```

Comment:
I'm curious - should it be `type dhcp_host`? we'll need to push this into Charrua's Dhcp_server.Config.t (and/or modify that type).

Reply:
Hm, yes, I think `type dhcp_host` is nicer. I chose `type dhcp_config` because in the C code it's `struct dhcp_config {}`.